google/cloud/bigquery/table.py

```diff
@@ -1673,16 +1674,22 @@ def to_dataframe(
                   progress bar as a graphical dialog box.
 
                 ..versionadded:: 1.11.0
-            create_bqstorage_client (bool):
-                Optional. If ``True`` (default), create a BigQuery Storage API
-                client using the default API settings. The BigQuery Storage API
+            create_bqstorage_client Optional[bool]:
+                If ``True`` (default), create a BigQuery Storage API client
+                using the default API settings. The BigQuery Storage API
                 is a faster way to fetch rows from BigQuery. See the
                 ``bqstorage_client`` parameter for more information.
 
                 This argument does nothing if ``bqstorage_client`` is supplied.
 
                 ..versionadded:: 1.24.0
 
+            date_as_object Optional[bool]:
```

Comment:
Parameter type should be enclosed in parentheses, otherwise it confuses the parser and it ends up looking weird in the docs:

![Screenshot from 2020-06-29 08-33-40](https://user-images.githubusercontent.com/1235663/85981486-8e577600-b9e4-11ea-91c9-aef565a5fa30.png)


Applies to multiple places, please check.